base-helm-configs/memcached/memcached-helm-overrides.yaml

```diff
@@ -539,30 +632,31 @@ volumePermissions:
   ## Init container' Security Context
   ## Note: the chown of the data folder is done to containerSecurityContext.runAsUser
   ## and not the below volumePermissions.containerSecurityContext.runAsUser
+  ## @param volumePermissions.containerSecurityContext.seLinuxOptions [object,nullable] Set SELinux options in container
   ## @param volumePermissions.containerSecurityContext.runAsUser User ID for the init container
   ##
   containerSecurityContext:
+    seLinuxOptions: {}
     runAsUser: 0
-
 ## Prometheus Exporter / Metrics
 ##
 metrics:
   ## @param metrics.enabled Start a side-car prometheus exporter
   ##
-  enabled: false
+  enabled: true
```

Comment:
we have external exporters for this. This also breaks if labs aren't installing prometheus first which many don't as it's not a req for testing other things. Can we keep this disabled. (we need to disable it in maria too tbh.)

Reply:
ok no problem I will disable it...Done